common/src/main/resources/schema/datasources_1_4.xsd

```diff
@@ -724,6 +724,17 @@
           </xs:documentation>
         </xs:annotation>
       </xs:element>
+      <xs:element name="fair" type="xs:boolean" minOccurs="0">
```

Comment:
One of the toString() had it after use-strict-min, most of them before -- just needs to be same for XSD, xml test cases and toString() impls